assets/RNAseq_report.Rmd

```diff
@@ -407,6 +1131,7 @@ The table for download can be found [here](./differential_gene_expression/final_
 Additional tables for all individual contrasts are provided [here](./differential_gene_expression/DE_genes_tables).
 
 <!-- Show table here removed as I dont like showing only top 20 genes.
+TODO Is this still necessary?
```

Comment:
If the table here is not shown (fine by me, please clarify with @qbicStefanC , then the text above should be adapted), as no list is shown.